readme.md

```diff
@@ -1,157 +1,138 @@
    ```
 
-   (or `just test-integrated`)
+   ```bash
+   ddn update data-connector-link sqlserver_connector --add-all-resources
+   ```
 
-## Write a database execution test
+   `--add-all-resources` flag adds all the models and commands present in the database to the connector metadata.
 
-1. Create a new file under `crates/ndc-sqlserver/tests/goldenfiles/<your-test-name>.json`
-2. Create a new test in `crates/ndc-sqlserver/tests/tests.rs` that looks like this:
-   ```rs
-   #[tokio::test]
-   async fn select_5() {
-       let result = common::test_query("select_5").await;
-       insta::assert_json_snapshot!(result);
-   }
-   ```
-3. Run the tests using `just dev`
-4. Review the results using `cargo insta review`
-
-## Write a SQL translation snapshot test
-
-1. Create a new folder under `crates/query-engine/tests/goldenfiles/<your-test-name>/`
-2. Create `request.json` and `tables.json` files in that folder to specify your
-   request
-3. Create a new test in `crates/query-engine/tests/tests.rs` that looks like this:
-   ```rs
-   #[tokio::test]
-   async fn select_5() {
-       let result = common::test_translation("select_5").await;
-       insta::assert_snapshot!(result);
-   }
+4. Create a build
+
+   ```bash
+   ddn build supergraph-manifest
    ```
-4. Run the tests using `just dev`
-5. Review the results using `cargo insta review`
 
-## Testing metrics
+   This will return information about the build:
+
+   |               |                                                                                                   |
+   | ------------- | ------------------------------------------------------------------------------------------------- |
+   | Build Version | bd96bb221a                                                                                        |
+   | API URL       | https://<PROJECT_NAME>-bd96bb221a.ddn.hasura.app/graphql                                          |
+   | Console URL   | https://console.hasura.io/project/<PROJECT_NAME>/environment/default/build/bd96bb221a/graphql     |
+   | Project Name  | <PROJECT_NAME>                                                                                    |
+   | Description   |                                                                                                   |
+
+   Follow the project configuration build [guide](https://hasura.io/docs/3.0/project-configuration/builds/) to apply
+   other actions on the build.
 
-We have a Prometheus / Grafana set up in Docker. Run `just start-metrics` to
-start them, you can then navigation to `localhost:3001` for Grafana, or
-`localhost:9090` for Prometheus.
+5. Test the API
 
-## Linting
+   The console URL in the build information cna be used to open the GraphiQL console to test out the API
 
-Run `just lint` to run clippy linter
+## Contributing
 
-run `just lint-apply` to attempt to autofix all linter suggestions
+ndc-sqlserver is still in early stages of development and we are currently not accepting contributions.
```

Comment:
Why?? We should accept all contributions right @gneeri ?

Reply:
Yeah, I think if someone were willing to put in the time, we'd be happy for the contribution. Perhaps we could just say something like:

`ndc-sqlserver is still in early stages and we have some work actively underway, but we are always happy for any community contributions.`